.travis.yml

```diff
@@ -91,6 +91,8 @@ script:
   - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then ./python Tools/scripts/patchcheck.py --travis $TRAVIS_PULL_REQUEST; fi
   # `-r -w` implicitly provided through `make buildbottest`.
   - make buildbottest TESTOPTS="-j4 -uall,-cpu"
+  # Run `make smelly` to check if Python leaks any symbols
+  - make smelly
```

Comment:
Oh, "make smelly" exits with an exit code 0 which means "success" even if libpython leaks a symbol :-( The change isn't as simple as I expected, sorry.

I wrote PR #4057 which reimplements "make smelly" in Python to get a non-zero exit code if libpython leaks a symbol.

Reply:
Oh yes, I was looking for a workaround for the same. 

I'd pick something else now. Thanks :)